content/06.data-resource-management.md

```diff
@@ -66,10 +66,8 @@ Keeping a read-only copy of raw data alongside a workflow as well multiple backu
 This also removes the imperative of storing intermediate files as these can be easily regenerated by the workflow.
 
 When sharing or storing files and results, data version control can keep track of differences in files such as changes from tool parameters or versions.
-The version control tools discussed in the [Workflow-based project management](## workflow-based-project-management) section are primarily designed to handle small files, but repositories such as the Open Science Framework, Figshare, Zenodo, and Dryad can be used for storing larger files and datasets.
-
-The Open Science Framework (OSF; [@doi:10.5195/JMLA.2017.88]) is a free service that provides powerful collaboration and sharing tools, provides built-in version control, integrates with other storage and version control repositories, guarantees data preservation, and enables you to keep projects private until they are ready to share. Like other services geared towards data sharing, OSF also enables generation of a digital object identifier (doi) for each project.
-While other services such as Git Large File Storage (LFS), Figshare [@url:https://figshare.com/], Zenodo [@url:https://zenodo.org/], and the Dryad Digital Repository [@url:https://datadryad.org/] each provide important services for sharing and version control, OSF provides the most comprehensive set of free tools for managing data storage and backup.
+The version control tools discussed in the [Workflow-based project management](## workflow-based-project-management) section are primarily designed to handle small files, but repositories such as the Open Science Framework (OSF), Git Large File Storage (LFS), Figshare, Zenodo, and Dryad can be used for storing larger files and datasets [@doi:10.5195/JMLA.2017.88; @url:https://git-lfs.github.com/; @url:https://figshare.com/; @url:https://zenodo.org/; @url:https://datadryad.org/].
```

Comment:
Git LFS is not a repository, it's a technique - that is supported by GitHub :).

So I guess we could say GitHub's support for LFS, or something?